<?xml version="1.0" encoding="utf-8"?>
<launch>

  <!--node name="joint_interface" pkg="baxter_interface" type="joint_trajectory_action_server.py" output="screen"/>
  <include file="$(find baxter_kinematics)/launch/move_group_mimick_node.launch" />
  <include file="$(find baxter_moveit_config)/launch/baxter_grippers.launch" /-->

  <rosparam command="load" file="$(find baxter_kinematics)/src/parameters.yaml" />

  <!-- Simulation -->
  <!--node name="get_sim_eef_pose_node" pkg="baxter_kinematics" type="get_sim_eef_pose_service" output="screen" args="joint_states:=/robot/joint_states"/-->

  <!-- Simulation and Real robot-->
  <node name="move_to_position_node" pkg="baxter_kinematics" type="move_to_position_service" output="screen" args="joint_states:=/robot/joint_states" />

  <node name="execute_trajectory_node" pkg="baxter_kinematics" type="execute_trajectory_service" output="screen" args="joint_states:=/robot/joint_states"/>

  <node name="restart_robot_node" pkg="baxter_kinematics" type="restart_robot_service" output="screen" args="joint_states:=/robot/joint_states" />

  <!-- Real robot--> 
  <!--node name="get_real_eef_pose_service" pkg="baxter_kinematics" type="get_real_eef_pose_service" output="screen" /-->

  <!-- if="$(arg real_robot)" -->

</launch>
